defs/scenarios/pacemaker/pacemaker_node1_found.yaml

```diff
@@ -0,0 +1,20 @@
+checks:
+  node1-found:
+    requires:
+      property: core.plugins.pacemaker.PacemakerBase.offline_nodes
+      op: contains
+      value: node1
```

Comment:
I tried running this against a node that had pacemaker installed but not configured and it failed because offline_nodes returns None if there aren't any. Instead you should have it return [] so that this always works.

Reply:
Fixed on c82b72d1a3be6ba00b6d2de421383f4c853ac395